waywiser_agrarsense/ros_bridge/build_docker_container.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Set default values
+DOCKER_IMAGE_NAME="agrarsense-ros1-bridge:latest"
+DOCKERFILE_DIR="src/WayWiseR/waywiser_agrarsense/ros_bridge"
+DOCKER_CONTAINER_NAME="agrarsense_ros_bridge"
```

Comment:
Not sure this is the best default, because it makes assumptions on where WayWiseR is located and where the script is called from. I think it is OK to assume that the docker file is in the same directory as this script, so sth like below might be better?

```suggestion
DOCKERFILE_DIR="$(dirname "$0")"
DOCKER_CONTAINER_NAME="agrarsense_ros_bridge"
```